test/unittests/t_util.cc

```diff
@@ -792,7 +792,10 @@ TEST_F(T_Util, RfcTimestamp) {
   time_t time1 = time(NULL);
   string str = RfcTimestamp();
   strptime(str.c_str(), format, &tm);
-  time_t time2 = mktime(&tm);
+  time_t time2 = mktime(&tm) - timezone;
+  if (tm.tm_isdst > 0) {
```

Comment:
Sorry, I wasn't clear in my comment.  I am not sure if extra handling of daylight saving is necessary.  But it might.  So did you test if it is necessary?